src/Core/Services/Implementations/UserService.cs

```diff
@@ -356,6 +361,124 @@ public async Task<bool> VerifyTwoFactorEmailAsync(User user, string token)
                 "2faEmail:" + email, token);
         }
 
+        public async Task<CredentialCreateOptions> StartWebAuthnRegistrationAsync(User user)
+        {
+            await _u2fRepository.DeleteManyByUserIdAsync(user.Id);
```

Comment:
This cleans out all old u2f keys when user starts using webauthn?

Reply:
Whops, good catch.

Reply:
It should only clean out the pending register/login key requests though.